boards/arm/disco_l475_iot1/disco_l475_iot1_defconfig

```diff
@@ -22,6 +22,7 @@ CONFIG_CLOCK_CONTROL=y
 # Default: PLL driven by HSI
 CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
 CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
+#CONFIG_CLOCK_STM32_SYSCLK_SRC_MSI=y
```

Comment:
Remove this line (same as the one below)

Reply:
DONE, available with new update